aspnetcore/grpc/configuration.md

```diff
@@ -37,12 +37,13 @@ The following table describes options for configuring gRPC channels:
 
 | Option | Default Value | Description |
 | ------ | ------------- | ----------- |
-| `HttpClient` | New instance | The `HttpClient` used to make gRPC calls. A client can be set to configure a custom `HttpClientHandler`, or add additional handlers to the HTTP pipeline for gRPC calls. By default a new `HttpClient` instance is created. |
+| `HttpClient` | New instance | The `HttpClient` used to make gRPC calls. A client can be set to configure a custom `HttpClientHandler`, or add additional handlers to the HTTP pipeline for gRPC calls. By default a new `HttpClient` instance is created for the channel. It will automatically be disposed. |
```

Comment:
```suggestion
| `HttpClient` | New instance | The `HttpClient` used to make gRPC calls. A client can be set to configure a custom `HttpClientHandler`, or add additional handlers to the HTTP pipeline for gRPC calls. By default, a new `HttpClient` instance is created for the channel. It will automatically be disposed. |
```